Assets/MRTK/Core/Inspectors/Profiles/MixedRealityControllerMappingProfileInspector.cs

```diff
@@ -15,6 +15,18 @@ namespace Microsoft.MixedReality.Toolkit.Input.Editor
     [CustomEditor(typeof(MixedRealityControllerMappingProfile))]
     public class MixedRealityControllerMappingProfileInspector : BaseMixedRealityToolkitConfigurationProfileInspector
     {
+        private struct ControllerMappingSignature
```

Comment:
Could this be made [a `readonly struct`](https://docs.microsoft.com/en-us/dotnet/csharp/write-safe-efficient-code#declare-readonly-structs-for-immutable-value-types)? I'm not sure it's a super necessary thing, but I'm pretty sure it helps the compiler in some cases by saving it from making defensive copies of value types when it knows the type can't be changed. Something I was reading up on recently and this seems like a decent spot for it!